components/00-base/_variables.components.scss

```
@@ -681,7 +681,7 @@ $ct-attachment-dark-wrapper-background-color: ct-color-dark('background') !defau
 $ct-back-to-top-space-bottom: ct-spacing(8) !default;
 $ct-back-to-top-space-right: ct-spacing(2) !default;
 $ct-back-to-top-background-color: ct-color-light('interaction-background') !default;
-$ct-back-to-top-border-color: $ct-back-to-top-background-color !default;
+$ct-back-to-top-border-color: ct-color-light('background-light') !default;
```

@AlexSkrypnyk If the goal is to set to the same as the background as back to top - shouldn't it be 
```suggestion
$ct-back-to-top-border-color: ct-color-light('interaction-background') !default;
```

---

No, please read the description

---

> this means that the border will not be visible on the light page content but will be in a contrast on footer.

---

My solution makes the border subtle and visually noticeable only on dark footer